lib/quadquizaminos_web/views/layout_view.ex

```diff
@@ -1,3 +1,17 @@
 defmodule QuadquizaminosWeb.LayoutView do
   use QuadquizaminosWeb, :view
+
+  defp admin_dropdown do
```

Comment:
Great work separating this from the html template

Reply:
Thanks